SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -117,6 +116,7 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 	});
 
 	LocalDeploymentManager->Init(GetOptionalExposedRuntimeIP());
+	SpatialGDKEditorInstance = FModuleManager::GetModuleChecked<FSpatialGDKEditorModule>("SpatialGDKEditor").GetSpatialGDKEditorInstance();
```

Comment:
The SpatialGDkEditor and SpatialGDKEditorToolbar have the same loading phase, is there a guarantee the editor module will be loaded before this one?
![image](https://user-images.githubusercontent.com/31517089/82670261-b52bcb00-9c34-11ea-9543-14f34a428081.png)

Reply:
With the current implementation, modules that have the same loading phase will be loaded in the order they're specified in the file.
I would be fine with adding comment on this line that we expect `SpatialGDKEditorModule` to be loaded here, just in case this changes when upgrading the engine version.